content/docs/api-reference/assets-api/index.md

```diff
@@ -28,14 +28,14 @@ Define the asset parameters and store in `/path/to/jsonfile`:
 ```json
 {
   "attributes": {
-    "arc_attachments": [
-      {
-        "arc_attachment_identity": "blobs/1754b920-cf20-4d7e-9d36-9ed7d479744d",
-        "arc_display_name": "Picture from yesterday",
-        "arc_hash_alg": "sha256",
-        "arc_hash_value": "01ba4719c80b6fe911b091a7c05124b64eeece964e09c058ef8f9805daca546b"
-      }
-    ],
+    "attachment": {
```

Comment:
I think it may be confusing to people to have the name of this attribute being "attachment": it should be a more meaningful name. Bear in mind that you can have many attachments and they are separately access controlled so the name of this key in the dictionary is as important as the name of any other text attribute.

Please change this appropriately in all instances

Reply:
Changed